community/yambar/build

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh -e
+
+meson \
+	--prefix=/usr \
+	-Dbackend-x11=disabled \
+	-Dbackend-wayland=enabled \
+	-Dplugin-mpd=disabled \
+	build
+DESTDIR="$1" ninja -C build install
```

Comment:
Please follow the style guide: https://kisscommunity.bvnf.space/kiss/style-guide/#0600

Reply:
Thanks, I hadn't see this page

Reply:
No worries. The style guide is also in this repo (style-guide.txt)